api/bolt/migrator/migrator.go

```diff
@@ -301,5 +305,12 @@ func (m *Migrator) Migrate() error {
 		}
 	}
 
+	if m.currentDBVersion < 23 {
```

Comment:
Add the following comment on top of this condition:

```golang
// Portainer 1.24.0-dev
```

Reply:
done